### PR TITLE
created crud services for app configs table

### PR DESCRIPTION
**Pull request by gadirovakhadija**

### CODE REVIEW

File: src/main/resources/mybatis/mappers/AppConfigsMapper.xml
Comment (hamidsultanzadeh) [inline src/main/resources/mybatis/mappers/AppConfigsMapper.xml:36]: please fix the typo

File: src/main/java/com/webperside/courseerpbackend/models/mybatis/appConfigs/AppConfigs.java
Comment (hamidsultanzadeh) [inline src/main/java/com/webperside/courseerpbackend/models/mybatis/appConfigs/AppConfigs.java:13]: it would be great to name it as AppConfig, and also be sure you are renamed all related classes

**Comment by hamidsultanzadeh** [comment]
@gadirovakhadija just some request changes here. 
after fixing that, you can merge it

**Review by hamidsultanzadeh — APPROVED** [review]

File: src/main/java/com/webperside/courseerpbackend/models/mybatis/appConfigs/AppConfigs.java
Comment (gadirovakhadija) [reply src/main/java/com/webperside/courseerpbackend/models/mybatis/appConfigs/AppConfigs.java:13]: We use app-configs naming in migration file. Can I also change migration file like app-config?

File: src/main/resources/mybatis/mappers/AppConfigsMapper.xml
Comment (aytiqaqash) [reply src/main/resources/mybatis/mappers/AppConfigsMapper.xml:36]: Waiting for this fix, to approve it.

File: src/main/java/com/webperside/courseerpbackend/models/mybatis/appConfigs/AppConfigs.java
Comment (hamidsultanzadeh) [reply src/main/java/com/webperside/courseerpbackend/models/mybatis/appConfigs/AppConfigs.java:13]: no, we don't need that. 
It's only for entity models. let me clarify that, if we store every app config data on each row on app_configs table, it means we'll get config data in this entity. it would be nice to name it like AppConfig. because every entity will contain only one row, and it indicates that, this app config entity only for one config value

**Review by hamidsultanzadeh — APPROVED** [review]

**Comment by hamidsultanzadeh** [comment]
@aytiqaqash please leave your decision
@gadirovakhadija After getting 2 approve, feel free to merge it to main branch

**Review by aytiqaqash — CHANGES_REQUESTED** [review]
The only change is from Builder to SuperBuilder.

**Review by aytiqaqash — APPROVED** [review]
LGTM

**Pull request merged**
